priv/cabbage/apps/itest/lib/account.ex

```diff
@@ -79,9 +94,36 @@ defmodule Itest.Account do
     {:ok, address}
   end
 
+  defp hash(message), do: ExthCrypto.Hash.hash(message, ExthCrypto.Hash.kec())
+
   defp create_account_from_secret(secret, passphrase) do
-    Ethereumex.HttpClient.request("personal_importRawKey", [secret, passphrase], [])
+    if Application.get_env(:cabbage, :reorg) do
+      with_retries(fn ->
+        Ethereumex.HttpClient.request("personal_importRawKey", [secret, passphrase], url: "http://localhost:9000")
+      end)
+
+      with_retries(fn ->
+        Ethereumex.HttpClient.request("personal_importRawKey", [secret, passphrase], url: "http://localhost:9001")
+      end)
+    else
+      with_retries(fn ->
+        Ethereumex.HttpClient.request("personal_importRawKey", [secret, passphrase], [])
+      end)
+    end
```

Comment:
remind me why do we do this again? is it so that we unlock accounts on both nodes?

Reply:
yes

Reply:
is there an alternative way of achieving the same thing? this looks kinda brute force 🗡️